src/target/riscv/riscv-011.c

```diff
@@ -425,6 +425,8 @@ static dbus_status_t dbus_scan(struct target *target, uint16_t *address_in,
 		.out_value = out,
 		.in_value = in
 	};
+	if (address_in)
+		*address_in = 0;
```

Comment:
There's a caller which prints out address_in in an error message, and the compiler complains that the value is uninitialized when printed. Since this is riscv-011 which no one is testing, I prefer to keep the change as small as possible.